packages/orbit-design-tokens/output/docs-tokens.json

```diff
@@ -14164,28 +14164,6 @@
       "value": "rgb(186, 199, 213)"
     }
   },
-  "modalBorderRadiusMobile": {
```

Comment:
Unlike the tailwind package, a token deletion **is a breaking change**, even if it is just an internal token. I would just mark this as deprecated instead of removing it. We can (and should) do a cleanup of all the tokens soon™.

The tailwind class removal can be kept, as it is undocumented 👍

Reply:
Agree, updated.